module/VuFind/src/VuFind/Autocomplete/SolrCN.php

```diff
@@ -60,11 +60,12 @@ public function setConfig($params)
     /**
      * Process the user query to make it suitable for a Solr query.
      *
-     * @param string $query Incoming user query
+     * @param string $query   Incoming user query
+     * @param array  $options Array of extra parameter
      *
-     * @return string       Processed query
+     * @return string        Processed query
```

Comment:
Since $options is intentionally unused here, we should add an annotation to prevent PHPMD from complaining:

```suggestion
     * @return string        Processed query
     *
     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
```